README.md

```diff
@@ -31,6 +31,11 @@ contributing to a specific part of the project, please join the appropriate
 special interest group (SIG). Details for all of these items are below. We are
 a friendly, collaborative group and look forward to working together!
 
+Technical committee members, maintainers, and approvers are encouraged
+to list their [areas of interest in this
+document](areas-of-interest.md) to help community members find
```

Comment:
I suggest adding a link to areas-of-interest.md to the TOC, "Members' areas of interest".

Reply:
I've added a section to the TOC.